package.json

```diff
@@ -37,6 +37,7 @@
     "cli-table": "https://github.com/telerik/cli-table/tarball/v0.3.1.2",
     "color": "3.0.0",
     "colors": "1.1.2",
+    "detect-newline": "^2.1.0",
```

Comment:
It is safer to use an exact version. You can use `npm i detect-newline --save-exact`. Same goes for `stringify-package` dependency.